wordpress-mobile/WordPress-Android
# Save media capture path during orientation changes

Fixes #11296

### Description

This PR saves the media capture path when the activity is destroyed, and restores it when it is recreated. This allows the processing of a captured photo to resume when the activity returns to the foreground after an orientation change has taken place. See https://github.com/wordpress-mobile/WordPress-Android/issues/11296#issuecomment-1057672316 for more details.

#### To test:

1. Start in Portrait mode.
2. Add an Image block and tap "Take a Photo" from the Choose image options.
3. Once the camera app has loaded, turn the device to Landscape mode.
4. Tap the on-screen button to take the photo, still in Landscape.
5. Tap the on-screen "OK" button to accept the photo, still in Landscape.

The photo should upload and be displayed correctly in the editor.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
